server/events.py

```diff
@@ -146,3 +152,66 @@ class EventResponse(EventRequest):
     """
 
     status_code: StatusCode
+
+
+class EventHandler:
+    """An request event handler."""
+
+    def __init__(self, client: Client, connection: ConnectionManager):
+        """Initializes the event handler for each client.
+
+        Args:
+            client: The client sending the requests.
+            connection: The connection to the room.
+        """
+        self.client = client
+        self.connection = connection
```

Comment:
I would rename this as `manager`: a connection basically corresponds to a client.